### ISSUE TITLE
python extension conflict

### ISSUE THREAD

**Issue by SuperDasher** [Darkempire78/Discord-Tools#33]
almost all of the internal chats inside vscode don't work because somehow it conflicts with the python extension from Microsoft. I tested it with every other extension and apparently only that one creates problems

**Comment by Darkempire78** [comment]
That's very weird, because it's working very well for me.
Can you give me more about this issue ? @SuperDasher

**Comment by SuperDasher** [comment]
I tried to look into it more and I saw that the problem was about another extension linked to the Python one from Microsoft. The ID of the extension is littlefoxteam.vscode-python-test-adapter if you want to look more into it. It seems that uninstalling it made all problems go away, sorry for my misunderstanding of the issue

**Comment by Darkempire78** [comment]
No problem and thank you for that it's interresting

**Comment by SuperDasher** [comment]
hey, sorry for disturbing again about this issue, but I found out more about it.

So, the extension works just fine in any scenario, except when I open any python script that makes the Python extension enable itself. After that, whenever I try to open any channel on the sidebar, this is what happens
![image](https://user-images.githubusercontent.com/72823509/143769527-b60fce05-ce5c-4488-9268-502552d471d1.png)

The only way for it to work again is to either deactivate the extension, or have it activated without any python script on the screen making the Python extension activate
![image](https://user-images.githubusercontent.com/72823509/143769795-fba91894-f758-4f87-a0d4-f026d8b9a17b.png)
![image](https://user-images.githubusercontent.com/72823509/143769899-9af6c0a2-96de-4ddc-b30d-7f3222835e84.png)

And even while working without the python extension activated, there are still some channels that don't work either way, but I think that could also be because I'm part of 100 servers

tl;dr : I confirmed that the Python extension by Microsoft causes some conflicts
